src/Core/Services/Implementations/CollectionService.cs

```diff
@@ -124,9 +124,9 @@ public async Task<IEnumerable<Collection>> GetOrganizationCollections(Guid organ
             }
 
             IEnumerable<Collection> orgCollections;
-            if (await _currentContext.OrganizationAdmin(organizationId))
+            if (await _currentContext.OrganizationAdmin(organizationId) || await _currentContext.OrganizationCustom(organizationId))
```

Comment:
This is only checking whether or not the user has the `type` of `custom`. This needs to go a step further and evaluate the permissions encompassed within the `Manage All Collections` parent node (Create New Collections, Edit Any Collection, Delete Any Collection). At the moment, I'm not sure of the exact cases needed and will be posting on the task for clarification.

Reply:
That makes sense. I tested all of the possible permission configurations and it seemed to output the desired result but I will wait for further clarification.

Reply:
```suggestion
            if (await _currentContext.OrganizationAdmin(organizationId) || await _currentContext.ViewAllCollections(organizationId))
```

This will cover both use cases coming from the web vault. Both situations are guarded on the client side before making the `API` call.